### PR TITLE
Add OrientDB example

### PR DESCRIPTION
**Pull request by pmantica1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ python -m graphql_compiler.tool <input_file.graphql >output_file.graphql
 It's modeled after Python's `json.tool`, reading from stdin and writing to stdout.
 
 ## Table of contents
-  * [FAQ](#faq)
+  * [Features](#features)
+  * [Code Sample](#code-sample)
```
Comment (bojanserafimov) [inline README.md]: link doesn't work

File: README.md [hunk 2]
```diff
@@ -56,64 +57,68 @@ It's modeled after Python's `json.tool`, reading from stdin and writing to stdou
...
+## Features
+* **Databases and Query Languages:** We currently support a single database, OrientDB version 2.2.28+, and two query languages that OrientDB supports: the OrientDB dialect of gremlin, and OrientDB's own custom SQL-like query language that we refer to as MATCH, after the name of its graph traversal operator. With OrientDB, MATCH should be the preferred choice for most users, since it tends to run faster than gremlin, and has other desirable properties. See the Execution model section for more details.
    
    Support for relational databases including PostgreSQL, MySQL, SQLite,
    and Microsoft SQL Server is a work in progress. A subset of compiler features are available for
    these databases. See the [SQL](#sql) section for more details.
+* **GraphQL Language Features:**  We prioritized and implemented a subset of all functionality supported by the GraphQL language. We hope to add more functionality over time.
 
-**Q: Do you plan to support other databases / more GraphQL features in the future?**
+## OrientDB Example
```
Comment (bojanserafimov) [inline README.md]: Please explain what the example does. Nobody is motivated to read a big piece of code for the sake of reading it.

File: README.md [hunk 3]
```diff
@@ -56,64 +57,68 @@ It's modeled after Python's `json.tool`, reading from stdin and writing to stdou
...
+```python3
+from graphql_compiler import (
+    get_graphql_schema_from_orientdb_schema_data, graphql_to_match
+)
+from graphql_compiler.schema_generation.utils import ORIENTDB_SCHEMA_RECORDS_QUERY
+from graphql_compiler.tests.conftest import init_integration_graph_client
 
-A: We'd love to, and we could really use your help! Please consider contributing to this project
```
Comment (bojanserafimov) [inline README.md]: why is this line removed?

Comment (pmantica1) [reply README.md]: It's included in the FAQ.
